[iOS] Fix Service JSON parsing (#97)
Polidea/blemulator_flutter: CHANGELOG.md

## 1.0.1

> * iOS - Fix Service JSON parsing

Reviewer: ```suggestion
* Fix issue where accessing characteristic's properties were null on iOS
```

> * Fix issue where accessing characteristic's properties were null on iOS

Reviewer: ```suggestion
* Fix an issue causing characteristics' properties to be null on iOS
```
